pkg/model/keycloak_deployment.go

```diff
@@ -157,8 +158,9 @@ func KeycloakDeployment(cr *v1alpha1.Keycloak) *v13.StatefulSet {
 								},
 							},
 							ReadinessProbe: &v1.Probe{
+								InitialDelaySeconds: 30,
 								TimeoutSeconds:      1,
-								InitialDelaySeconds: 10,
+								FailureThreshold:    10,
```

Comment:
Failure threshold 10 should be ok in most cases, but what happens when we're doing an upgrade and migration code slows startup down?